framework_tvb/tvb/interfaces/web/templates/jinja2/base_template.html

```diff
@@ -47,6 +47,7 @@
         <script type="text/javascript" src="/static/jquery/jquery-ui.js?{{ currentVersionJS }}"></script> <!--JQuery custom imports for the Range, Slider controller -->
         <script type="text/javascript" src="/static/jquery/jquery.json.js"></script>
         <script type="text/javascript" src="/static/js/genericTVB.js?{{ currentVersionJS }}"></script>
+        <script type="text/javascript" src="/static/js/filters.js?{{ currentVersionJS }}"></script>
```

Comment:
This import place might be too high.
Have you tried to import this script in `genericAdapterFormFields.html` and in `main_burst.html` ?

Reply:
If I put the inclusion in these two + spatial/includedResources.html (for the two stimulus types) then it's allright.